src/Chunk.ts

```diff
@@ -869,13 +865,13 @@ export default class Chunk {
 	): string {
 		const hash = createHash();
 		hash.update(
-			[addons.intro, addons.outro, addons.banner, addons.footer].map(addon => addon || '').join(':')
+			[addons.intro, addons.outro, addons.banner, addons.footer].map(addon => addon ?? '').join(':')
```

Comment:
Turns out the fallback is actually not needed (and the `Addons` type can be adapted accordingly). I will push a small fix to your branch so that we can merge this.